### PR TITLE
Create getting_started.md

### PR DESCRIPTION
**Pull request by gvelez17**
# Quick docs for hackathon

## Description

How to use the current version of composedb

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+```
+
+
+```
+
+* Compile the schema to [format]
+
+This creates the models on ceramic:
+
+`composedb composite:create [graphql_def_file]`
```
Comment (youngnishant) [inline docs/getting_started.md:16]: Do i have to install compose db globally?

File: docs/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    content: String! @length(max:2000)
+}
+
+```
+
+* generate a DID key 
+
+If you do not already have a DID key handy, you will need to create one 
+
+If you have downloaded and installed this repo or the [js-glaze](https://github.com/ceramicstudio/js-glaze) repo you can run
```
Comment (youngnishant) [inline docs/getting_started.md:24]: there is inbuilt function to create did in composeDB itself 'composedb did:COMMAND'

File: docs/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Getting Started with ComposeDB
+
+## Create a schema
+
+* Write a graphQL compatible composite schema in a text file, such as
+
+example: task.schema
```
Comment (youngnishant) [inline docs/getting_started.md:7]: it should be schema.graphql something

File: docs/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+* generate a DID key 
+
+If you do not already have a DID key handy, you will need to create one 
+
+If you have downloaded and installed this repo or the [js-glaze](https://github.com/ceramicstudio/js-glaze) repo you can run
+```
+cd packages/cli
+node bin/run.js did:generate-seed
```
Comment (youngnishant) [inline docs/getting_started.md:27]: it is `node bin/run.js did:create`

File: docs/getting_started.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+```
+save this result for use as the did key in the next step.
+
+* Create schema and output encoded composite
+
+`composedb composite:create [graphql_def_file] -k [didkey] -o [output_file.json]`
```
Comment (youngnishant) [inline docs/getting_started.md:33]: gives error "SyntaxError: Non-base16 character" when this cmd is used "composedb composite:create schema.graphql -k  did:key:z6MkrYP78xpdBEWsba2k4kHzPc3NTqJGuV7MRKdN7EEQNjPu -o composite.json"
Screenshot 2022-07-26 at 7 44 38 PM

Comment (youngnishant) [inline docs/getting_started.md:33]: USAGE
  $ composedb composite:create INPUT

ARGUMENTS
  INPUT  a path to file containing valid ceramic composite definition in GraphQL Schema Definition Language

OPTIONS
  -c, --ceramic-url=ceramic  Ceramic API URL
  -k, --did-private-key=key  DID Private Key
  -o, --output               a path to file where the resulting encoded composite definition should be saved
```

File: docs/getting_started.md (hunk 1 above)
Comment (BuchananQuantum) [reply docs/getting_started.md]: Yes. Just install the turborepo

**Comment by PaulLeCam** [comment]
@gvelez17 @3avi this is probably outdated by now, shall we close it?

**Comment by gvelez17** [comment]
yes closing thx

**Pull request closed without merging**